autospec/specfiles.py

```diff
@@ -324,7 +342,24 @@ def write_files(self):
             self._write("\n%files {}\n".format(pkg))
             self._write("%defattr(-,root,root,-)\n")
             for filename in sorted(self.packages[pkg]):
-                self._write("{}\n".format(filename))
+                # Separate any directive prefix
+                # This will break if your directive contains a slash!
+                line = filename.split("/", 1)
+                if len(line) < 1:
+                    print_fatal("Filenames must start with a / ({0})".format(filename))
+                    sys.exit(1)
+                if len(line) > 1:
+                    # Write the prefix unmodified
+                    self._write("{}".format(line[0]))
+                    # Filename is remaining part
+                    filename = line[1]
+                else:
+                    filename = line[0]
+                if ' ' in filename:
```

Comment:
Need to handle any whitespace, including tabs

Reply:
Will fix